### DISCUSSION TITLE
How does this even work?

### DISCUSSION THREAD

**Discussion by abundant-ray** [copy/v86#522] [Q&A]
How can you know enough about javascript to get it to run operating systems? I dont understand how you can write some code to run os's on completely unsupported hardware. the ability to run stuff like windows 98 on a chromebook is great :)

**Comment by TechStudent10** [comment]
https://github.com/copy/v86#api-examples

**Comment by ReconEvacuateDEVX** [comment]
browsers have become more powerful in recent years and people have become smarter (or sometimes dumber) so this is very much possible

**Comment by ReconEvacuateDEVX** [comment]
the first counter strike can be run in full on a browser also along with more graphical interactive things
